### ISSUE TITLE
App crashes when navigating from YouTube component page to another page and back.

### ISSUE THREAD

**Issue by CPhilipse** [inProgress-team/react-native-youtube#427]
For some reason when I go from the page where the YouTube component is initiated like below to another page and then back to the page where the YouTube component is initiated, the app crashes for no reason. The component and all works, it shows the video. So I don't see any reason for the app to crash, the app doesn't crash when commenting this component out. Does anyone else share the same issue? Is there a solution for this?
`<YouTube
                    videoId="qYyEG9ioWqU" // The YouTube video ID
                    play={false} // control playback of video with true/false
                    fullscreen={false} // control whether the video should play in fullscreen or inline
                    loop // control whether the video should loop when ended
                    onReady={e => this.setState({ isReady: true })}
                    onChangeState={e => this.setState({ status: e.state })}
                    onChangeQuality={e => this.setState({ quality: e.quality })}
                    onError={e => this.setState({ error: e.error })}
                    style={{ alignSelf: 'stretch', height: 300 }}
                    apiKey="api_key"/>`

**Comment by o5faruk** [comment]
I am having the same issue, mind saying how you solved it?

**Comment by CPhilipse** [comment]
@o5faruk I didn't solve it sadly, I worked around it. It's because of my code structure that I could, basically what I did was when I go to another page from the Youtube page, that when I go back I am on the home page of videos, so not the same page with the Youtube component. I worked around it by adding an addlistener of react navigation. And say when going to another page from the page with the Youtube component, change in the state 'home' to true which is the home page of videos (which shows all posts). When 'home' is false, you are on the post with the Youtube component. But I got too much trouble with the component anyhow, so I replaced it with a webview. I closed it because I didn't see an option to delete it.
